plugins/course-apps/wiki/messages.js

```diff
@@ -26,8 +26,7 @@ const messages = defineMessages({
   },
   enablePublicWikiHelp: {
     id: 'course-authoring.pages-resources.wiki.enable-public-wiki.help',
-    defaultMessage: `If enabled, edX users can view the course wiki even when
-    they're not enrolled in the course.`,
+    defaultMessage: 'If enabled, any registered user can view the course wiki even if not enrolled in the course',
```

Comment:
```suggestion
    defaultMessage: 'If enabled, any registered user can view the course wiki even if they are not enrolled in the course',
```

Reply:
How strongly do you feel about this? Both versions convey the same message, but the one in the PR is shorter and avoids using "they" in the singular use. While English is clearly moving to accept such usage, I consider it best to avoid the usage when using it adds no clarity.

If you feel strongly about this, I'll make the change.